flipt-php/src/Flipt/Client/FliptClient.php

```diff
@@ -19,18 +19,20 @@ final class FliptClient
     protected string $entityId;
     protected array $context;
     protected LoggerInterface $logger;
+    protected array $headers = [];
 
     /**
      * @param array<string, string> $context
      */
-    public function __construct(string|Client $host, string $namespace = "default", array $context = [], string $entityId = '', AuthenticationStrategy $authentication = null)
+    public function __construct(string|Client $host, string $namespace = "default", array $context = [], string $entityId = '', AuthenticationStrategy $authentication = null, array $headers = [])
```

Comment:
We need to document that headers should be associative array in phpdoc. At least GuzzleHttp\Client expects that format of array